apprise/plugins/ntfy.py

```diff
@@ -778,6 +789,9 @@ def url(self, privacy=False, *args, **kwargs):
         if self.__tags:
             params["tags"] = ",".join(self.__tags)
 
+        if self.__actions is not None:
```

Comment:
Almost done; this needs to change to `if self.__actions:` .... drop `is not None`